### ISSUE TITLE
Where to place model?

### ISSUE THREAD

**Issue by Ph0rk0z** [gitmylo/audio-webui#6]
I have downloaded the model separately but I'm not sure on the folder structure. I tried putting it in suno--bark but it still tries to download the model.

**Comment by gitmylo** [comment]
Bark models are saved in `~/.cache/suno/bark_v0` by default. Where `~` is `C:\Users\username` on windows.
I might change it later to use the models directory from the webui. But I didn't yet because bark works slightly differently and I'll have to monkeypatch more bark things.

**Comment by Ph0rk0z** [comment]
I got it working and wow. When it wants to it clones better than tortoise. Faster too. Just unfortunately there is no consistency and the next sentence can be another voice. I set the wav temp to 1 and it would get back almost indistinguishable copies.

**Comment by gitmylo** [comment]
Right, bark can switch voices, this is also possible with the official speaker prompt files

**Comment by Ph0rk0z** [comment]
I'm having good luck with .8 temperature.. for that new RVC, any recommendations for models, esp RVC model. I see the TTS models are pretty much coqui.

**Comment by gitmylo** [comment]
> I'm having good luck with .8 temperature.. for that new RVC, any recommendations for models, esp RVC model. I see the TTS models are pretty much coqui.

some that i know of
https://huggingface.co/QuickWick/Music-AI-Voices/tree/main
https://discord.gg/aihub
